### PR TITLE
change severity of `no-console` rule to `error` to clean up eslint output

### PR DESCRIPTION
**Pull request by B2o5T**
explicitly disable them with `eslint-disable-next-line`

### CODE REVIEW

File: examples/monaco-graphql-webpack/src/index.ts [hunk 1]
```diff
@@ -320,7 +321,8 @@ export function renderGithubLoginButton() {
       { provider: 'github', scope: ['user'] },
       async (err: Error, data: { token: string }) => {
         if (err) {
-          console.error('Error Authenticating with GitHub: ' + err);
+          // eslint-disable-next-line no-console
+          console.error('Error authenticating with GitHub:', err);
```
Comment (B2o5T) [inline examples/monaco-graphql-webpack/src/index.ts:325]: here (and for other below cases) concatenation of `error` can produce unexpected `[object Object]` value

File: .eslintrc.js [hunk 2]
```diff
@@ -304,10 +304,9 @@ module.exports = {
     },
     {
       // Resources are typically our helper scripts; make life easier there
-      files: ['resources/*.js', '**/resources/*.js'],
+      files: ['resources/**', 'scripts/**'],
```
Comment (B2o5T) [inline .eslintrc.js:307]: allow `console` for scripts also

File: .eslintrc.js [hunk 3]
```diff
@@ -304,10 +304,9 @@ module.exports = {
     },
     {
       // Resources are typically our helper scripts; make life easier there
-      files: ['resources/*.js', '**/resources/*.js'],
+      files: ['resources/**', '**/resources/**', 'scripts/**'],
```
Comment (B2o5T) [inline .eslintrc.js:307]: allow `console` for `scripts` folder also

File: packages/vscode-graphql-execution/src/extension.ts [hunk 4]
```diff
@@ -23,14 +23,14 @@ function getConfig() {
 }
 
 export function activate(context: ExtensionContext) {
-  console.log('activated!');
```
[the remark this answers is not in the document]
Comment (acao) [reply packages/vscode-graphql-execution/src/extension.ts:26]: this is just an embarassing oversight haha, case in point for this rule

File: packages/monaco-graphql/src/GraphQLWorker.ts [hunk 5]
```diff
@@ -48,6 +48,7 @@ export class GraphQLWorker {
       );
       return graphqlDiagnostics.map(toMarkerData);
     } catch (err) {
+      // eslint-disable-next-line no-console
       console.error(err);
```
Comment (acao) [inline packages/monaco-graphql/src/GraphQLWorker.ts:52]: thanks yeah, can replace these with a proper logger of some kind eventually

**Review by acao — APPROVED** [review]

**Pull request merged**